dev/make-models.py

```diff
@@ -67,8 +67,8 @@
 
     for classname, class_version in superclasses:
         cls = f.file.class_named(classname, class_version)
-        print(cls.class_code)  # noqa: T001
-        print(  # noqa: T001
+        print(cls.class_code)
```

Comment:
I don't know why this one is not triggering, actually. It should, it seems to me.

Reply:
Ahh, I figured it out. I'll fix.